rmgpy/tools/data/sim/mbSampled/simulate.log

```diff
@@ -0,0 +1 @@
+Error: library reaction i2 <=> i1 is pressure dependent, and its high-pressure limit rate could not be extracted. Ignoring this library reaction (the rate could still be estimated from families).
```

Comment:
Is this log file needed? It looks like the simple example and liquid example only have `chem.inp`, `input.py`, and `species_dictionary.txt`, but maybe this is used somewhere and I just didn't notice